diff -up netpbm-10.35.51/editor/pamcomp.c.pamcomp netpbm-10.35.51/editor/pamcomp.c
--- netpbm-10.35.51/editor/pamcomp.c.pamcomp	2008-09-18 13:45:15.000000000 +0200
+++ netpbm-10.35.51/editor/pamcomp.c	2008-09-18 14:03:19.000000000 +0200
@@ -325,25 +325,6 @@ validateComputableHeight(int const origi
 
 
 static void
-validateComputableHeight(int const originTop, 
-                         int const overRows) {
-
-    if (originTop < 0) {
-        if (originTop < -INT_MAX)
-            pm_error("Overlay starts too far above the underlay image to be "
-                     "computable.  Overlay can be at most %d rows above "
-                     "the underlay.", INT_MAX);
-    } else {
-        if (INT_MAX - originTop <= overRows)
-            pm_error("Too many total rows involved to be computable.  "
-                     "You must have a shorter overlay image or compose it "
-                     "higher on the underlay image.");
-    }
-}
-
-
-
-static void
 computeOverlayPosition(int                const underCols, 
                        int                const underRows,
                        int                const overCols, 
