--- netpbm-10.35/converter/other/pnmtotiffcmyk.c.glibc	2006-09-18 12:20:06.000000000 +0200
+++ netpbm-10.35/converter/other/pnmtotiffcmyk.c	2007-08-23 09:18:30.000000000 +0200
@@ -974,8 +974,8 @@ int main( int argc, char **argv ) {
 
   if ( (err = parseOpts( argc, argv, rt )) ) goto exit ;
   
-  if ( (err = rt->in->open( rt->in, rt )) ) goto exit ;
-  if ( (err = rt->out->open( rt->out, rt )) ) goto exit ;
+  if ( (err = (rt->in->open)( rt->in, rt )) ) goto exit ;
+  if ( (err = (rt->out->open)( rt->out, rt )) ) goto exit ;
 
   while ( rt->in->hasMore( rt->in ) ) {
     if ( (err = rt->in->next( rt->in, &r, &g, &b )) ) goto exit ;
diff -up netpbm-10.35.62/converter/ppm/xvminitoppm.c.glibc netpbm-10.35.62/converter/ppm/xvminitoppm.c
--- netpbm-10.35.62/converter/ppm/xvminitoppm.c.glibc	2009-04-13 09:14:06.000000000 +0200
+++ netpbm-10.35.62/converter/ppm/xvminitoppm.c	2009-04-14 12:27:48.000000000 +0200
@@ -52,7 +52,7 @@ parseCommandLine(int const argc,
 
 
 static void
-getline(FILE * const ifP,
+get_line(FILE * const ifP,
         char * const buf,
         size_t const size) {
 
@@ -105,7 +105,7 @@ readXvHeader(FILE *         const ifP,
     int rc;
     bool endOfComments;
     
-    getline(ifP, buf, sizeof(buf));
+    get_line(ifP, buf, sizeof(buf));
 
     if (!STRNEQ(buf, "P7 332", 6))
         pm_error("Input is not a XV thumbnail picture.  It does not "
@@ -113,14 +113,14 @@ readXvHeader(FILE *         const ifP,
 
     endOfComments = FALSE;
     while (!endOfComments) {
-        getline(ifP, buf, sizeof(buf));
+        get_line(ifP, buf, sizeof(buf));
         if (STRNEQ(buf, "#END_OF_COMMENTS", 16))
             endOfComments = TRUE;
         else if (STRNEQ(buf, "#BUILTIN", 8))
             pm_error("This program does not know how to "
                      "convert builtin XV thumbnail pictures");
     }
-    getline(ifP, buf, sizeof(buf));
+    get_line(ifP, buf, sizeof(buf));
     rc = sscanf(buf, "%u %u %u", &cols, &rows, &maxval);
     if (rc != 3)
         pm_error("error parsing dimension info '%s'.  "
