--- netpbm-10.31/converter/other/xwdtopnm.c.xwdfix	2006-02-14 15:13:58.000000000 +0100
+++ netpbm-10.31/converter/other/xwdtopnm.c	2006-02-14 15:15:37.000000000 +0100
@@ -985,7 +985,7 @@ getpix(pixelReader * const rdrP) {
             unsigned long const bitsToTakeMask = lsbmask[nBitsToTake];
                 /* E.g. if nbitsToTake is 4, this is 0x0000000F */
 
-            unsigned long bitsToTake;
+            unsigned int bitsToTake;
 
             assert(nBitsToTake <= 32);
             
