diff -up netpbm-10.35.40/pm_config.in.h.rgbtxt netpbm-10.35.40/pm_config.in.h
--- netpbm-10.35.40/pm_config.in.h.rgbtxt	2008-03-09 11:28:32.000000000 +0100
+++ netpbm-10.35.40/pm_config.in.h	2008-03-14 12:27:28.000000000 +0100
@@ -67,9 +67,9 @@
 #define RGB_DB2 "PBMplus_Dir:RGB.TXT"
 #define RGB_DB3 "PBMplus_Dir:RGB.TXT"
 #else
-#define RGB_DB1 "/usr/lib/X11/rgb.txt"
-#define RGB_DB2 "/usr/share/X11/rgb.txt"
-#define RGB_DB3 "/usr/X11R6/lib/X11/rgb.txt"
+#define RGB_DB1 "/usr/share/netpbm/rgb.txt"
+#define RGB_DB2 "/usr/lib/X11/rgb.txt"
+#define RGB_DB3 "/usr/share/X11/rgb.txt"
 #endif
 
 /* CONFIGURE: This is the name of an environment variable that tells
