diff -up netpbm-10.35.58/converter/ppm/ximtoppm.c.ximtoppmsegfault netpbm-10.35.58/converter/ppm/ximtoppm.c
--- netpbm-10.35.58/converter/ppm/ximtoppm.c.ximtoppmsegfault	2009-01-28 18:37:05.000000000 +0100
+++ netpbm-10.35.58/converter/ppm/ximtoppm.c	2009-01-28 19:01:14.000000000 +0100
@@ -50,6 +50,10 @@ parseCommandLine(int argc, char ** argv,
     OPTENT3(0,   "alphaout",   OPT_STRING, 
             &cmdlineP->alpha_filename, &alphaoutSpec, 0);
 
+    opt.opt_table = option_def;
+    opt.short_allowed = FALSE;
+    opt.allowNegNum = FALSE;
+
     optParseOptions3(&argc, argv, opt, sizeof(opt), 0);
         /* Uses and sets argc, argv, and all of *cmdlineP. */
 
