diff --git a/editor/ppmfade b/editor/ppmfade
index 5091651..23b3ecc 100755
--- a/editor/ppmfade
+++ b/editor/ppmfade
@@ -113,6 +113,9 @@ for ($n = 0; $n < @ARGV; $n++) {
         $mode = $BLOCK;
     } elsif ("$ARGV[$n]" eq "-mix") {
         $mode = $MIX;
+    } elsif ($ARGV[$n] eq "-help" || $ARGV[$n] eq "--help" || $ARGV[$n] eq "-h") {
+        print "ppmfade: Use 'man ppmfade' for help.\n";
+        exit 1;
     } else {
         print "Unknown argument: $ARGV[$n]\n";
         exit 100;
