diff -up netpbm-10.35.67/buildtools/configure.pl.configfix netpbm-10.35.67/buildtools/configure.pl
--- netpbm-10.35.67/buildtools/configure.pl.configfix	2009-09-04 13:56:59.000000000 +0200
+++ netpbm-10.35.67/buildtools/configure.pl	2009-09-04 14:16:48.000000000 +0200
@@ -1893,7 +1893,7 @@ if ($platform eq "GNU") {
         # We don't know what to do here.  We used to (before 10.20) just
         # just assume the compiler was gcc.  We know that the gcc stuff
         # above does NOT work for HP native compiler.
-        push(@config_mk, "LDSHLIB =\n");
+        push(@Makefile_config, "LDSHLIB =\n");
     }
 } elsif ($platform eq "AIX") {
     push(@Makefile_config, 'LDFLAGS = -L /usr/pubsw/lib', "\n");
